### PR TITLE
refactor: migrate to composite run

### PR DESCRIPTION
**Pull request by SweetOps**

### CODE REVIEW

**Comment by SweetOps** [comment]
@takezyou @haya14busa , could you please do review

File: script.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+curl -sfL https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b "${TEMP_PATH}" "${REVIEWDOG_VERSION}" 2>&1
+echo '::endgroup::'
+
+echo '::group:: Installing ansible-lint ... https://github.com/ansible/ansible-lint'
+pip3 install --no-cache-dir ansible-lint=="${INPUT_ANSIBLELINT_VERSION}"
+echo '::endgroup::'
+
+export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
+
+echo '::group:: Running tflint with reviewdog 🐶 ...'
```
Comment (haya14busa) [inline script.sh:18]:
```suggestion
echo '::group:: Running ansible-lint with reviewdog 🐶 ...'
```

**Review by haya14busa — COMMENTED** [review]
Can you update the README about `ansiblelint_version` argument?

**Comment by haya14busa** [comment]
@takezyou Can you take a look? LGTM other than my comments.

**Comment by takezyou** [comment]
@SweetOps 
No problem elsewhere!
Please fix the comment part.

**Comment by SweetOps** [comment]
@haya14busa, I've fixed all things that you asked

**Review by takezyou — APPROVED** [review]
@SweetOps LGTM!

**Pull request merged**